src/extension.ts

```diff
@@ -13,17 +13,20 @@ async function checkLean4(): Promise<boolean> {
     if (folders) {
         folderPath = folders[0].uri.fsPath
     }
-    const cmd = `${executablePath()} --version`
+    
+    const env = addServerEnvPaths(process.env);
+    const cmd = `${executablePath()}`
```

Comment:
```suggestion
    const cmd = executablePath()
```
Or am I missing something?

Reply:
You're not missing anything, no.